plutus-playground-client/src/Editor/State.purs

```diff
@@ -24,9 +26,12 @@ initialState =
     pure
       $ State
           { keyBindings
-          , feedbackPaneMinimised: false
+          , feedbackPaneMinimised: true
           , lastCompiledCode: Nothing
           , currentCodeIsCompiled: false
+          , feedbackPaneDragStart: Nothing
+          , feedbackPaneExtend: 0
+          , feedbackPanePreviousExtend: 0
```

Comment:
I'm just wondering if these three things naturally group together so tightly that they warrant their own type... 🤔 

What do you think?

Reply:
Maybe. But we're out of time, so I vote merging as is and revisiting this later.